NewStyle.css

```diff
@@ -0,0 +1,149 @@
+.B{
+    position:relative;
+    top:-600px;
+    right:-25px;
+    font-size: 300%;
+    color: rgb(5, 5, 5);
+    display: inline-block;
+     
+  }
+  .name {
+      /* text-align: justify; */
+    display: flex;
+  justify-content: space-between;
+}
+#A{
+    position:relative;
+    top: -500px;
+        }
+#R{
+    position:relative;
+    top: -400px;
+        }
+ #L{
+    position:relative;
+    top: -300px;
+        }
+ #Y{
+     position:relative;
+    top: -200px;
+        }
```

Comment:
these all have `position: relative`, so you could move that styling to the common class that all the letters have